docs/content/docs/tutorials/emr-on-eks-example.md

```diff
@@ -85,13 +85,13 @@ Now we can go ahead and install EMR on EKS controller. First, let's export envir
 ```
 export SERVICE=emrcontainers
 export RELEASE_VERSION=`curl -sL https://api.github.com/repos/aws-controllers-k8s/$SERVICE-controller/releases/latest | grep '"tag_name":' | cut -d'"' -f4`
-export ACK_SYSTEM_NAMESPACE=ack-system
+export ACK_K8S_NAMESPACE=ack-system
```

Comment:
Any reason to rename this to `ACK_K8S_NAMESPACE`? i believe it is named the ACK system namespace elsewhere too..

Reply:
I tried to keep same name as mentioned in [IRSA](https://aws-controllers-k8s.github.io/community/docs/user-docs/irsa/#create-an-iam-role-for-your-ack-service-controller) setup. There seems to be regression issue bcoz both names are being used in tutorials

Reply:
IMO It's the IRSA setup guide that should change. `ACK_SYSTEM_NAME` is what have been used previously (with CARM and other feature)